### ISSUE TITLE
Nano on terminal not showing cursor

### ISSUE THREAD

**Issue by devbnj** [microsoft/vscode#84284]
Issue Type: <b>Bug</b>

Nano on terminal not showing cursor. Connected to ec2 instance on AWS. Working fine till the update just now. 

VS Code version: Code 1.40.0 (86405ea23e3937316009fc27c9361deee66ffbf5, 2019-11-06T17:09:34.601Z)
OS version: Darwin x64 18.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz (4 x 2500)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|10.00GB (2.36GB free)|
|Process Argv|-psn_0_94231|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
solidity-extended|bea|3.0.2
vscode-markdownlint|Dav|0.31.2
jshint|dba|0.10.21
vscode-eslint|dba|1.9.1
vscode-coffeescript-preview|dre|1.0.1
tslint|eg2|1.0.44
terraform-autocomplete|erd|0.0.8
beautify|Hoo|1.5.0
composer-support-client|Hyp|0.19.12
solidity-solhint|idr|1.1.4
solidity|Jua|0.0.64
javascript-test-runner|leg|1.0.0
vscode-azureterraform|ms-|0.3.0
python|ms-|2019.10.44104
azure-account|ms-|0.8.7
Go|ms-|0.11.9
proto|pet|0.0.2
java|red|0.52.0
vscodeintellicode|Vis|1.2.0
vscode-java-debug|vsc|0.23.0
vscode-java-dependency|vsc|0.6.0
vscode-java-pack|vsc|0.8.0
vscode-java-test|vsc|0.21.0
vscode-maven|vsc|0.19.1

</details>

**Comment by Tyriar** [comment]
Can you post a screenshot (or better yet a gif) and repro steps? Does the cursor show before opening nano?

**Comment by devbnj** [comment]
Hi Tyriar. It was working fine. And the cursor is there before nano. Funny thing the cursor appears scrolling horizontally, but disappears while using the up and down arrow keys. Here's the screen shot. 
![Screen Shot 2019-11-08 at 1 16 56 PM](https://user-images.githubusercontent.com/11469142/68500735-671c2080-022a-11ea-9a72-fcc36ce22b1e.png)

![Screen Shot 2019-11-08 at 1 17 29 PM](https://user-images.githubusercontent.com/11469142/68500749-6f745b80-022a-11ea-8a7a-2cd223b7c3fb.png)

**Comment by Tyriar** [comment]
Can you enable escape sequence logging and see what the output when you press up/down and left/right is? https://github.com/microsoft/vscode/wiki/Terminal-Issues#enabling-escape-sequence-logging

I'm wondering if nano is telling the terminal to hide the cursor or something else is going on.

**Comment by devbnj** [comment]
Hey Tyriar, Which log file should I be copying here. I started code --log trace and pressing (func) F1, I get to 'Developers: Open Log File..'. I know the logical is Window. There is nothing there - its a clean slate. I went to the terminal window and used the up/down and sideways keys. Nothing showed up on the logs. Image attached.

I switched to the Main Log. Image attached.
![Screen Shot 2019-11-08 at 7 06 31 PM](https://user-images.githubusercontent.com/11469142/68518690-1a9d0900-025b-11ea-84ae-0f79dc511170.png)

![Screen Shot 2019-11-08 at 7 07 05 PM](https://user-images.githubusercontent.com/11469142/68518702-1f61bd00-025b-11ea-966b-ecc22d3fa08d.png)
